frontend/public/components/app.jsx

```diff
@@ -202,6 +202,7 @@ class App extends React.PureComponent {
             <LazyRoute path="/k8s/cluster/clusterrolebindings/:name/edit" exact kind="ClusterRoleBinding" loader={() => import('./RBAC' /* webpackChunkName: "rbac" */).then(m => m.EditRoleBinding)} />
             <LazyRoute path="/k8s/ns/:ns/:plural/:name/attach-storage" exact loader={() => import('./storage/attach-storage' /* webpackChunkName: "attach-storage" */).then(m => m.AttachStorage)} />
 
+            <LazyRoute path="/k8s/ns/:ns/:plural/new/form" exact loader={() => import('./storage/create-pvc' /* webpackChunkName: "create-pvc" */).then(m => m.CreatePvc)} />
```

Comment:
This route needs to be specific to create PVC. It can't match all types.

Reply:
Likely because you're using `connectToPlural` when you don't need to. Just use the `PersistentVolumeClaim` model directly. You know what type you're creating.